docs/doc/14-sql-commands/00-ddl/91-catalog/_category_.json

```diff
@@ -0,0 +1,3 @@
+{
+  "label": "Catalog"
```

Comment:
It looks we can put the catalog to the first:
<img width="460" alt="image" src="https://github.com/datafuselabs/databend/assets/172204/dddf682a-6e7e-40e7-b614-0592321a7c65">